pkg/store/sqlite/transactions.go

```diff
@@ -131,6 +131,9 @@ func (s *Store) UpdateTransaction(ctx context.Context, t *models.Transaction) (e
 	// Update modified timestamp (in place).
 	t.Modified = time.Now()
 
+	// Update last update timestamp.
+	t.LastUpdate = sql.NullTime{Time: time.Now(), Valid: true}
```

Comment:
```suggestion
	// NOTE: do not update `LastUpdate` timestamp - this refers to when a secure envelope is sent/received.
```

Reply:
When I made updates to a transaction, the last update timestamp didn't change in the db. It always remained NULL.

Reply:
To follow up on my last message, I assume the timestamp never changed because when I created a transaction the status was draft instead of pending.